src/enums/credit-card.ts

```diff
@@ -2,7 +2,7 @@ export enum CreditCard {
     AmericanExpress = 'amex',
     Argencard = 'argencard',
     Cabal = 'cabal',
-    Consosud = 'censosud',
```

Comment:
thats a breaking change, we need a different version here 👍🏻

Reply:
yeah, 0.4.0 should be good enough 👍🏻

Reply:
bumped